Source/WebCore/platform/ScrollSnapAnimatorState.h

```diff
@@ -98,7 +111,8 @@ class ScrollSnapAnimatorState {
 
     void transitionToUserInteractionState();
     void transitionToDestinationReachedState();
-
+    void preserveCurrentTargetForAxis(ScrollEventAxis axis);
+    void setFocusedElementForAxis(ScrollEventAxis axis);
```

Comment:
To make style bot happy:
```suggestion
    void preserveCurrentTargetForAxis(ScrollEventAxis);
    void setFocusedElementForAxis(ScrollEventAxis);
```

Reply:
Fixed.